### PR TITLE
espeak-ng: initial package

### PR DESCRIPTION
**Pull request by nuive**

### CODE REVIEW

**Comment by nuive** [comment]
Same as Pull Request https://github.com/LibreELEC/LibreELEC.tv/pull/5786/ but for the master branch

File: packages/mediacenter/kodi/package.mk [hunk 1]
```diff
@@ -62,6 +62,10 @@ configure_package() {
     KODI_PULSEAUDIO="-DENABLE_PULSEAUDIO=OFF"
   fi
 
+  if [ "$ESPEAK_SUPPORT" = yes ]; then
+    PKG_DEPENDS_TARGET="$PKG_DEPENDS_TARGET espeak-ng"
```
Comment (chewitt) [inline packages/mediacenter/kodi/package.mk:66]: PKG_DEPENDS_TARGET+=" espeak-ng"

File: packages/audio/espeak-ng/package.mk [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+PKG_NAME="espeak-ng"
+PKG_VERSION="1.50"
+PKG_SHA256="80ee6cd06fcd61888951ab49362b400e80dd1fac352a8b1131d90cfe8a210edb"
+PKG_LICENSE="GPL"
+PKG_SITE="https://github.com/espeak-ng/espeak-ng"
+PKG_URL="https://github.com/espeak-ng/espeak-ng/releases/download/$PKG_VERSION/$PKG_NAME-$PKG_VERSION.tgz"
+PKG_DEPENDS_HOST="gcc:host "
+PKG_DEPENDS_TARGET="toolchain espeak-ng:host"
+PKG_LONGDESC="eSpeak NG is an open source speech synthesizer that supports more than hundred languages and accents"
```
Comment (chewitt) [inline packages/audio/espeak-ng/package.mk:12]: "more than one hundred" or "more than a hundred" not "more than hundred" .. bad English, but i'm nitpicking :)

**Comment by nuive** [comment]
All fixed. I'd forgotten to fix the "than [a] hundred"( I copied the description directly from eSpeak's GitHub).

**Comment by chewitt** [comment]
Please squash to a single commit, then it's ready to merge.

**Comment by nuive** [comment]
Done.

**Review by chewitt — APPROVED** [review]
Thanks!

**Comment by HiassofT** [comment]
LE builds are broken since this PR:
```
UNPACK      espeak-ng
    FIXCONFIG      /home/hias/private/libreelec/libreelec-master/build.LibreELEC-RPi4.arm-11.0-devel/build/espeak-ng-1.50/
BUILD      espeak-ng (host)
    TOOLCHAIN      configure
make[1]: Entering directory '/home/hias/private/libreelec/libreelec-master/build.LibreELEC-RPi4.arm-11.0-devel/build/espeak-ng-1.50'
Makefile:2423: warning: ignoring prerequisites on suffix rule definition
CDPATH="${ZSH_VERSION+.}:" && cd . && /bin/bash /tmp/espeak-ng/missing aclocal-1.15 -I m4
/bin/bash: /tmp/espeak-ng/missing: No such file or directory
make[1]: *** [Makefile:713: aclocal.m4] Error 127
make[1]: Leaving directory '/home/hias/private/libreelec/libreelec-master/build.LibreELEC-RPi4.arm-11.0-devel/build/espeak-ng-1.50'
FAILURE: scripts/build espeak-ng:host during pre_configure (package.mk)
*********** FAILED COMMAND ***********
make distclean
**************************************
FAILURE: scripts/build espeak-ng:host has failed!
```
We are also seeing this on our Jenkins, see eg https://build.libreelec.net/blue/organizations/jenkins/Generic/detail/Generic/1302/pipeline/

**Comment by SupervisedThinking** [comment]
https://github.com/LibreELEC/LibreELEC.tv/blob/master/packages/audio/espeak-ng/package.mk#L16-L18

**Pull request merged**